src/store/macros/getters.ts

```diff
@@ -40,37 +40,31 @@ export const getters: GetterTree<MacrosState, RootState> = {
 
         return macro
       })
-      .sort((a, b) => a.name.localeCompare(b.name))
 
     return macros
   },
 
   // Gets visible macros, transformed. Should include the macro's config.
   // Is only used on the dashboard. Grouped by category.
   getVisibleMacros: (state, getters) => {
```

Comment:
This needed to be updated to utilize `getMacrosByCategory` so that macros would be sorted per-category, rather than being sorted overall then split into categories which lead to unexpected behaviour. Since I needed to update it anyway, I thought I'd refactor it to DRY things up somewhat

Reply:
Good point, this was actually rather stupid considering we will then do other operations on this!